hdfs3/core.py

```diff
@@ -7,7 +7,8 @@
 import warnings
 import fnmatch
 PY3 = sys.version_info.major > 2
-here = os.path.dirname(os.path.abspath(__file__))
+# here = os.path.dirname(os.path.abspath(__file__))
+here = '/usr/local/lib'
```

Comment:
Yes, it was intentional i think is more realistic to look for the `so` at `/usr/local/lib` and this is where its built in the container, but it should be configurable.